modules/ROOT/pages/desktop_release_notes.adoc

```diff
@@ -14,6 +14,36 @@
 
 toc::[]
 
+== Desktop App 5.3.0
+
+[discrete]
+=== General
+
+The ownCloud Desktop App is now available in version 5.3.0! +
+We strongly recommend updating the Desktop App to the latest available version so that you can access and experience a range of new features and security fixes. Users with an Enterprise subscription are entitled to receive dedicated support.
+
+Refer to the full change log for a list of bug fixes and changes at {desktop-releases-url}v5.3.0[GitHub, window=_blank].
+
+[discrete]
+=== Enhanced Functionality
+
+* Add support to provide a list of ports for the OAuth process
+* Support `Active Directory Federation Service` as identity provider
+
+[discrete]
+=== Notable Bugfixes
+
+ownCloud Desktop App 5.3.0 also delivers several important technical improvements such as:
+
+* Fix early-use crash in the folder watcher on Linux
+* Client stuck in `reconnecting`
+* Ensure folder are scheduled only once
```

Comment:
```suggestion
* Ensure folders are scheduled only once
```

I suppose this is a general statement about "folders" (plural).

Or it could be "Ensure folder is scheduled only once"

Reply:
I just copy/pasted from the desktop repo changelog.
I guess that you are fully correct and this needs also a change in the Desktop changelog.

Reply:
Off-topic, but why are we copying and pasting the changelog....
I thought we have https://owncloud.com/changelog/desktop/ for that.

Reply:
We have it here for all producs making the overview easier. In addition, we can (and do) write additional info like breaking changes and references to relevant doc content when and where necessary.

Reply:
I created a text fix for @phil-davis suggestion in the client repo too, see: https://github.com/owncloud/client/pull/11681